test/integration/tas/tas_test.go

```diff
@@ -979,6 +979,18 @@ var _ = ginkgo.Describe("Topology validations", func() {
 			ginkgo.Entry("invalid levels",
 				testing.MakeTopology("invalid-level").Levels([]string{"@invalid"}).Obj(),
 				testing.BeInvalidError()),
+			ginkgo.Entry("non-unique levels",
```

Comment:
It is not clear from this test case if the error is due to non-unique level, or hostname in the middle. 
Add another test case to show the error is thrown, eg.: block, block, hostname.

Reply:
Done